Gruntfile.coffee

```diff
@@ -33,25 +33,29 @@ module.exports = (grunt) ->
           'watch'
         ]
 
-    # Compass configuration
-    compass:
-      options:
-        require: [
-          'compass_sumo'
+    # Sass configuration
+    sass:
+      dev:
```

Comment:
Ik snap niet waarom je in uw targets steeds altijd alles twee keer wil specificeren. Als je als eerste "target" options neemt dan worden deze options overgenomen, en moet je dus enkel die specifieke overschrijven in je specifieke target.
Zie het stuk code dat je hier net hebt weggedaan.